klippy/extras/save_variables.py

```diff
@@ -0,0 +1,62 @@
+# Get and Save arbitrary variables in the config file so that values can be kept
+# across restarts.
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
+
+import sys,os,logging, ConfigParser
+
+error = ConfigParser.Error
```

Comment:
It's not a good idea to define global variables like this.  I recommend just using ConfigParser.Error explicitly in the locations where it is needed.

Reply:
I had used configfile.py as a base for my code. Have removed this now.